doc/03_Documents/01_Editables/28_Renderlet.md

```diff
@@ -71,7 +71,7 @@ Now editors are able to put elements onto the renderlet in the editmode.
 public function myGalleryAction(Request $request): array
 {
     if ('asset' === $request->query->get('type')) {
-        $asset = Asset::getById($request->query->getInt('id'));
+        $asset = Asset::getById(\Pimcore\Helper\ParameterBagHelper::getInt($request->query, 'id'));
```

Comment:
Should we really recommend this helper when writing new code? It's only for backward compatibility, right? So it doesn't make sense to add it to new code?